pyk/src/pyk/kore/rpc.py

```diff
@@ -69,11 +105,15 @@ def __exit__(self, *args: Any) -> None:
     def close(self) -> None: ...
 
     @abstractmethod
-    def command(self, bug_report_id: str, old_id: int, bug_report_request: str) -> list[str]: ...
+    def command(self, req_name: str, bug_report_request: str) -> list[str]: ...
 
     @abstractmethod
     def description(self) -> str: ...
 
+    @property
+    def _client_id(self) -> str:
```

Comment:
There's a single call to this, can be inlined.

Reply:
Fixed.